test/inventory_level_test.rb

```diff
@@ -0,0 +1,59 @@
+require 'test_helper'
+
+class InventoryLevelTest < Test::Unit::TestCase
+  def setup
+    super
+    @inventory_level_response = ActiveSupport::JSON.decode load_fixture('inventory_level')
+    @inventory_level = ShopifyAPI::InventoryLevel.new(@inventory_level_response['inventory_level'])
+  end
+
+  test 'Retrieve inventory levels for the specified inventory_items and locations' do
```

Comment:
Nitpick, but typically our test descriptions start lower case.

Normally we start with the method name that is being tested like:

```
test ".find with inventory_item_ids and location_ids returns expected inventory levels"
```

* `#` prefix for instance methods
* `.` prefix for class methods